compiler/rustc_target/src/spec/mod.rs

```diff
@@ -3166,7 +3166,7 @@ impl Target {
                 // Note that the `lp64e` is still unstable as it's not (yet) part of the ELF psABI.
                 check_matches!(
                     &*self.llvm_abiname,
-                    "lp64" | "lp64f" | "lp64d" | "lp64q" | "lp64e",
```

Comment:
lp64q is not listed anywhere else in the compiler and will ICE e.g. [here](https://github.com/rust-lang/rust/blob/0db9059726dbec82abdc21935f84a7f0e12ea495/compiler/rustc_codegen_ssa/src/back/metadata.rs#L355-L362), so it makes little sense to allow it in target specs.

Reply:
Thank you for noticing! That site can be changed to use `EF_RISCV_FLOAT_ABI_QUAD`

Reply:
Oh wow I did a slightly further investigation and it definitely *can't*, LLVM does a UB if you reach that branch lol, we have to die on it.